setup.py

```diff
@@ -26,13 +28,55 @@
 LONG_DESCRIPTION = readme
 AUTHOR = "Facebook AI Research"
 LICENSE = license
-REQUIREMENTS = (reqs.strip().split("\n"),)
+REQUIREMENTS = reqs.strip().split("\n")
+DEFAULT_EXCLUSION = ["test", "examples"]
+
+FULL_REQUIREMENTS = []
+# collect requirements.txt file in all subdirectories
+for file_name in glob.glob("**/*requirements.txt", recursive=True):
+    with open(file_name) as f:
+        reqs = f.read()
+        FULL_REQUIREMENTS.extend(reqs.strip().split("\n"))
+
+
+# overriding install and develop
+class OptionedCommand(object):
+    user_options = [("all", None, "include habitat_baselines in installation")]
+
+    def initialize_options(self):
+        super().initialize_options()
+        self.all = None
+
+    def run(self):
+        if not self.all:
+            DEFAULT_EXCLUSION.extend(
```

Comment:
Is there an option to specify instead `INCLUSION_ONLY` option? Most probably we don't want install anything else except `habitat` folder and don't want to update this lined with each new folder added.

Reply:
After some digging, I didn't find a more elegant way of doing this. `find_package` unfortunately does not have something similar to `INCLUSION ONLY`.